components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/util/EventHookHandlerUtils.java

```diff
@@ -0,0 +1,265 @@
+import org.wso2.carbon.identity.application.authentication.framework.model.AuthenticatedUser;
+import org.wso2.carbon.identity.application.common.model.Claim;
+import org.wso2.carbon.identity.application.common.model.ClaimMapping;
+import org.wso2.carbon.identity.application.common.model.User;
+import org.wso2.carbon.identity.core.ServiceURLBuilder;
+import org.wso2.carbon.identity.core.URLBuilderException;
+import org.wso2.carbon.identity.event.IdentityEventConfigBuilder;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.IdentityEventServerException;
+import org.wso2.carbon.identity.event.bean.ModuleConfiguration;
+import org.wso2.carbon.identity.event.event.Event;
+import org.wso2.identity.event.common.publisher.model.EventContext;
+import org.wso2.identity.event.common.publisher.model.EventPayload;
+import org.wso2.identity.event.common.publisher.model.SecurityEventTokenPayload;
+import org.wso2.identity.webhook.common.event.handler.constant.Constants;
+import org.wso2.identity.webhook.common.event.handler.internal.EventHookHandlerDataHolder;
+import org.wso2.identity.webhook.common.event.handler.model.EventData;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static com.wso2.identity.asgardeo.event.configuration.mgt.core.service.util.EventConfigurationMgtConstants.EVENT_CONFIG_SCHEMA_NAME_KEY;
+import static com.wso2.identity.asgardeo.event.configuration.mgt.core.service.util.EventConfigurationMgtConstants.ErrorMessages.ERROR_WHILE_CASTING_EVENT_CONFIG_AT_SERVER;
+import static org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils.CORRELATION_ID_MDC;
+
+/**
+ * This class contains the utility method implementations.
+ */
+public class EventHookHandlerUtils {
+
+    private static final Log log = LogFactory.getLog(EventHookHandlerUtils.class);
+
+    /**
+     * Get the identity property specified in identity-event.properties
+     *
+     * @param moduleName   The name of the module which the property belongs to
+     * @param propertyName The name of the property which should be fetched
+     * @return The required property
+     */
+    public static String getIdentityEventProperty(String moduleName, String propertyName) {
+
+        // Retrieving properties set in identity event properties
+        String propertyValue = null;
+        try {
+            ModuleConfiguration moduleConfiguration = IdentityEventConfigBuilder.getInstance()
+                    .getModuleConfigurations(moduleName);
+
+            if (moduleConfiguration != null) {
+                propertyValue = moduleConfiguration.getModuleProperties().getProperty(propertyName);
+            }
+        } catch (IdentityEventException e) {
+            log.error("An error occurred while retrieving module properties because " + e.getMessage(), e);
+        }
```

Comment:
Why we log and ignore this error ?

Reply:
Addressed with https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/1/commits/6e511b91f52c4b71ccff73c989894c76b285ebf0